pkg/cmd/pipeline/describe.go

```diff
@@ -131,10 +131,18 @@ func describeCommand(p cli.Params) *cobra.Command {
 			}
 
 			if len(args) == 0 {
-				err = askPipelineName(opts, p)
+				pipelineNames, err := pipeline.GetAllPipelineNames(p)
 				if err != nil {
 					return err
 				}
+				if len(pipelineNames) == 1 {
+					opts.PipelineName = pipelineNames[0]
+				} else {
+					err = askPipelineName(opts, p)
```

Comment:
Maybe we could pass the list of names to `askPipelineName` as an arg by adding a `[]string` param?
In `askPipelineName`, you could check if the list is empty to determine whether to make the call here to avoid doing it twice in some cases.